app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new
+    # temp = params[:comment]
+    text = params[:text]
+    @comment.text = text
+    user = current_user
+    @comment.user_id = user.id
+    @comment.post_id = params[:post_id]
+
+    redirect_to(request.env['HTTP_REFERER']) if @comment.save
```

Comment:
* Kindly add strong parameters here for example like this ▶️ `  
* `private `
   ` def post_params `
     ` params.require(:post).permit(:title, :text)`
   ` end`
* As This is a better security practice to help prevent accidentally allowing users to update sensitive model attributes.  [Refer back to lesson link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.)